packages/@aws-cdk/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -50,6 +50,21 @@ export enum LustreAutoImportPolicy {
   NEW_CHANGED_DELETED = 'NEW_CHANGED_DELETED',
 }
 
+/**
+  * The permitted Lustre data compression algorithms
+*/
+export enum LustreDataCompressionType {
+  /**
+  *
+  * `NONE` - (Default) Data compression is turned off when the file system is created.
+  */
+  NONE = 'NONE',
+  /**
+  * `LZ4` - Data compression is turned on with the LZ4 algorithm.
+  */
+  LZ4 = 'LZ4'
```

Comment:
Glad to see this instead of just an on/off here. It leaves us room for new compression algorithms in the future without breaking changes or weird contract updates.

Also, the hanging comma is missing here. I don't know why the linter didn't pick that up. Please run the linter before committing again. Granted, it may pick up a lot of code you didn't write if it's not picking this up.

Reply:
It seemed sensible since that's what the autogenerated L1 construct does as well, and I didn't want to put any constraints on it in the future.